src/internal/operators/bufferTime.ts

```diff
@@ -144,8 +146,14 @@ class BufferTimeSubscriber<T> extends Subscriber<T> {
       const timeSpanOnlyState = { subscriber: this, context, bufferTimeSpan };
       this.add(context.closeAction = scheduler.schedule(dispatchBufferTimeSpanOnly, bufferTimeSpan, timeSpanOnlyState));
     } else {
-      const closeState = { subscriber: this, context };
-      const creationState: DispatchCreateArg<T> = { bufferTimeSpan, bufferCreationInterval, subscriber: this, scheduler };
+      const closeState = { subscriber: this, context, destination: this.destination };
+      const creationState: DispatchCreateArg<T> = {
```

Comment:
I got a lint error about long line length. It's very distracting seeing red when you're trying to type, and it's just formatting. I am used to automatic code formatting. I hope this manual formatting I chose is ok. It seems odd to me in this day & age that we're not using something like Prettier to avoid subjective & tedious formatting decisions.